src/components/presentational/RotatingComponentCoordinator/RotatingComponentCoordinator.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { Children } from 'react';
+import { differenceInDays, differenceInMonths, differenceInWeeks } from 'date-fns';
+import isAfter from "date-fns/isAfter";
+
+export type RotatingComponentInterval = 'day' | 'week' | 'month';
+
+export interface RotatingComponentCoordinatorProps {
```

Comment:
The name of this component is misleading. Was expecting to see text rotating, or something to that effect.
This is showing specific text based on a the diff in date intervals.

Reply:
Happy to rename it, but I couldn't think of something better.  Feel free to suggest a new name.  😃

Reply:
I'll note that I had the same reaction to this name.  I'm also struggling to come up with an alternative name.  I'm fine with something verbose, but everything I come up with is *extremely* verbose.  This is a `TimeBasedComponentCarousel`?  It's not really a carousel because it has no paging controls though.  `TimeBasedComponentSwitcher`?  `TimeIntervalComponentSwitcher`?

Reply:
IDK...personally of the options I prefer RotatingComponentCoordinator...yeah it is a bit verbose but when I try to succinctly describe this thing to myself I end up saying that it's a rotation of content...

Reply:
TextDisplayByDateDifference ... ?

Reply:
It has a rotation of content that can be things other than just text, so probably don't want "Text" in the name.